easybuild/tools/module_naming_scheme/mns.py

```diff
@@ -126,6 +126,18 @@ def det_modpath_extensions(self, ec):
         # by default: an empty list of subdirectories to extend $MODULEPATH with
         return []
 
+    def det_user_modpath_extensions(self, ec):
```

Comment:
@geimer  @boegel This does not seem to be respecting `EASYBUILD_SUFFIX_MODULES_PATH` (which defaults to `all`):
```
conflict("GCCcore")
prepend_path("MODULEPATH", "/usr/local/software/jureca/Stages/Devel-2017a/modules/all/Compiler/GCCcore/5.4.0")
if isDir(pathJoin(os.getenv("HOME"), ".local/EasyBuild/modules/Compiler/GCCcore/5.4.0")) then
    prepend_path("MODULEPATH", pathJoin(os.getenv("HOME"), ".local/EasyBuild/modules/Compiler/GCCcore/5.4.0"))
end
```
Notice the missing `all` in the local path.